packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ReactViewGroupMapper.kt

```diff
@@ -0,0 +1,105 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay
+
+import android.annotation.SuppressLint
+import com.datadog.android.sessionreplay.internal.AsyncJobStatusCallback
+import com.datadog.android.sessionreplay.internal.recorder.MappingContext
+import com.datadog.android.sessionreplay.internal.recorder.mapper.BaseWireframeMapper
+import com.datadog.android.sessionreplay.internal.recorder.mapper.TraverseAllChildrenMapper
+import com.datadog.android.sessionreplay.model.MobileSegment
+import com.facebook.react.uimanager.Spacing
+import com.facebook.react.views.view.ReactViewBackgroundDrawable
+import com.facebook.react.views.view.ReactViewGroup
+
+internal class ReactViewGroupMapper :
+    BaseWireframeMapper<ReactViewGroup, MobileSegment.Wireframe>(),
+    TraverseAllChildrenMapper<ReactViewGroup, MobileSegment.Wireframe> {
+
+    @SuppressLint("VisibleForTests")
+    override fun map(
+        view: ReactViewGroup,
+        mappingContext: MappingContext,
+        asyncJobStatusCallback: AsyncJobStatusCallback
+    ): List<MobileSegment.Wireframe> {
+        val viewGlobalBounds = resolveViewGlobalBounds(
+            view,
+            mappingContext.systemInformation.screenDensity
+        )
+
+        val pixelDensity = mappingContext.systemInformation.screenDensity
+        val backgroundDrawable = view.background as? ReactViewBackgroundDrawable
+
+        val backgroundColor = backgroundDrawable?.color ?: 0
```

Comment:
Can you attach a screenshot of a view with no background color?
I'm not sure about setting `0` as default value here, I'm afraid it would make the background all black when it should be transparent by default

Reply:
good point. I'll make this null if there's no background drawable